README.md

```diff
@@ -4,6 +4,12 @@ The [etcd.io][] website, built using [Hugo][] and hosted on [Netlify][].
 
 ## Build prerequisites
 
+### Cloud build
+
+Visit [https://gitpod.io/#https://github.com/etcd-io/website](https://gitpod.io/#https://github.com/etcd-io/website) to launch a [Gitpod.io](https://gitpod.io) IDE that will allow you to build, preview and make changes to this repo.
```

Comment:
The cloud build doesn't have any "Build prerequisites" so it should be, for example, the first second-level heading in this page.